main/tasks/stratum_task.c

```diff
@@ -210,10 +210,33 @@ void stratum_task(void * pvParameters)
                     GLOBAL_STATE->extranonce_str = stratum_api_v1_message.extranonce_str;
                     GLOBAL_STATE->extranonce_2_len = stratum_api_v1_message.extranonce_2_len;
                 } else if (stratum_api_v1_message.method == CLIENT_RECONNECT) {
-                    ESP_LOGE(TAG, "Pool requested client reconnect...");
+                    ESP_LOGI(TAG, "Pool requested client reconnect to %s:%d (wait: %d seconds)",
+                        stratum_api_v1_message.new_host ? stratum_api_v1_message.new_host : "same host",
+                        stratum_api_v1_message.new_port ? stratum_api_v1_message.new_port : port,
+                        stratum_api_v1_message.wait_time);
+
+                    if (stratum_api_v1_message.new_host) {
```

Comment:
Should probably check that its actually a new url. Do we know if the new stratum url should persist?.